features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher-new/source/src/app/components/Apis/Details/Configuration/Configuration.jsx

```diff
@@ -355,84 +355,97 @@ class Configuration extends React.Component {
-                                            <Typography component='p' variant='body1'>
-                                                {api.version && <React.Fragment>{api.version}</React.Fragment>}
-                                            </Typography>
-                                            {/* Default Version */}
-                                            <Typography component='p' variant='subtitle2' className={classes.subtitle}>
-                                                <FormattedMessage
-                                                    id='Apis.Details.Configuration.Configuration.default.version'
-                                                    defaultMessage='Default Version'
-                                                />
-                                                <Tooltip
-                                                    placement='top'
-                                                    classes={{
-                                                        tooltip: classes.htmlTooltip,
-                                                    }}
-                                                    disableHoverListener
-                                                    title={
-                                                        <React.Fragment>
-                                                            <FormattedMessage
-                                                                id={'Apis.Details.Configuration.Configuration.' +
-                                                                'default.version.tooltip'}
-                                                                defaultMessage={'Marks one API version in a group as ' +
-                                                                'the default so that it can be invoked without ' +
-                                                                'specifying the version number in the URL. ' +
-                                                                'For example, if you mark ' +
-                                                                'http://host:port/youtube/2.0 as the default API, ' +
-                                                                'requests made to http://host:port/youtube/ are ' +
-                                                                'automatically routed to version 2.0. If you mark an ' +
-                                                                'unpublished API as the default, the previous  ' +
-                                                                'default published API will still be used as the ' +
-                                                                'until the new default API is published.'}
+                                            {/* Version */} {/* Default Version */}
+                                            { api.apiType === API.CONSTS.API &&
+                                                <React.Fragment>
```

Comment:
When conditionally rendering, shouldn't the content be wrapped with () if it's multiline?

Reply:
fixed with https://github.com/wso2/carbon-apimgt/commit/a5b4d09fd25478c04985b9e5f2eea62f56fed44d